src/OmniSharp.Host/project.json

```diff
@@ -5,7 +5,6 @@
   },
   "dependencies": {
     "OmniSharp.Abstractions": "1.0.0",
-    "OmniSharp.Nuget": "1.0.0",
```

Comment:
@david-driscoll: Please note this change. I think we need to invest in some effort to make this endpoint great again.